samples/openFABMAPcli.cpp

```diff
@@ -957,7 +961,7 @@ void drawRichKeypoints(const cv::Mat& src, std::vector<cv::KeyPoint>& kpts, cv::
 		colour = CV_RGB(255, 0, 0);
 	}
 	
-	for (int iii = kpts_sorted.size()-1; iii >= 0; iii--) {
+    for (int iii = (int)kpts_sorted.size()-1; iii >= 0; iii--) {
 
 		if (minResponse != maxResponse) {
 			normalizedScore = pow((kpts_sorted.at(iii).response - minResponse) / (maxResponse - minResponse), 0.25);

```

Comment:
Do you know why the response has an exponent of one quarter? Is this new or old?